src/components/WelcomeMat/components/WelcomeMatContent.js

```diff
@@ -0,0 +1,155 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import { stepType } from '../constants';
+import { Column, Grid } from '../../Grid';
+import { WelcomeMatStep } from './WelcomeMatStep';
+
+class WelcomeMatContent extends Component {
```

Comment:
this component can be done as a functional component. Can we use hooks in `react-lds`?

Reply:
Eventually yes, it would require a major version bump though as going from requiring `react@.6` to `react@.8` is a breaking change. We should def. do it in the near future though imo